src/TopStats.re

```diff
@@ -77,27 +86,36 @@ let make = (~title, ~resultsWithMap: temp_resultsWithRatingMap, ~startDate) =>
           (playerName, rating |> Js.Math.round |> int_of_float |> formatDiff)
         );
 
-    <Paper>
-      <div className="title">
-        <Typography variant="h6"> {text(title)} </Typography>
-      </div>
-      <div className="top-stats-container">
-        <SingleStatCard
-          playersWithStat=topWinPercentageRows
-          statName="Win Percentage"
-        />
-        <SingleStatCard
-          playersWithStat=topGoalsScoredPerMatchRows
-          statName="Goals Scored per Match"
-        />
-        <SingleStatCard
-          playersWithStat=topGoalsConcededPerMatchRows
-          statName="Goals Conceded per Match"
-        />
-        <SingleStatCard
-          playersWithStat=topEloDiffRows
-          statName="Elo Difference"
-        />
-      </div>
-    </Paper>;
+    switch (settingsQuery) {
+    | Loading => <CircularProgress />
+    | NoData
+    | Error(_) => <span> {text("Error")} </span>
```

Comment:
Same here, `NoData` should probably not be displayed as error?

Reply:
`NoData` is before we've started loading?
I haven't added "real" NoData/Error handling at all throughout the application. Created an issue for this: https://github.com/Yakimych/result-log/issues/74